util/ss58.ts

```diff
@@ -1,45 +1,27 @@
 import * as base58 from "../deps/std/encoding/base58.ts"
 import { Blake2b } from "../deps/wat_the_crypto.ts"
 
-// SS58PRE string (0x53533538505245 hex) encoded as Uint8Array
-const SS58PRE = Uint8Array.of(83, 83, 53, 56, 80, 82, 69)
-const CHECKSUM_LENGTH = 2
-const VALID_ADDRESS_LENGTHS: Record<number, boolean | undefined> = {
-  35: true,
-  36: true,
-  37: true,
-  38: true,
-}
-const VALID_PUBLIC_KEY_LENGTHS: Record<number, boolean | undefined> = {
-  32: true,
-  33: true,
-}
-
-export const encode = (
+export function encode(
   prefix: number,
   pubKey: Uint8Array,
   validNetworkPrefixes?: readonly number[],
-) => {
-  const encodeRawResult = encodeRaw(prefix, pubKey, validNetworkPrefixes)
-  if (encodeRawResult instanceof Error) return encodeRawResult
-  return base58.encode(encodeRawResult)
+): string {
+  return base58.encode(encodeRaw(prefix, pubKey, validNetworkPrefixes))
 }
+
 export const encodeRaw = (
```

Comment:
Lets make the rest of these `function` declarationss instead of arrow functions as well.